cdip_admin/api/v2/utils.py

```diff
@@ -319,7 +372,7 @@ def send_observations_to_routing(observations, gundi_ids):
                 # Send message to routing services
                 publisher.publish(
                     topic=settings.RAW_OBSERVATIONS_TOPIC,
-                    data=json.loads(msg_for_routing.json()),  # This is suboptimal. It's fixed in pydantic v2
+                    data=msg_for_routing.dict(),
```

Comment:
@marianobrc Just a question, how is this behaving now? Is the publisher doing some serialization of complex types? Or is this assuming scalars at this point?

Reply:
The publisher uses `json.dumps` with `default=str`, which is enough for serializing the most complex types we can expect at this point (datetime and UUID).